app/controllers/users_controller.rb

```diff
@@ -1,4 +1,6 @@
 class UsersController < ApplicationController
+  before_action :only_loggedin_users, only:[:index, :edit, :update, :destroy]
+  before_action :correct_user, only: [:edit, :update]
```

Comment:
![Screen Shot 2021-04-30 at 8 30 21 PM](https://user-images.githubusercontent.com/74635797/116695311-0589dc00-a9f3-11eb-8c59-6e149b2d8728.png)
correct_user method doesn't exist.

You have to make:
```
def correct_user
end
```
and place it after **user_params**